### PR TITLE
Adding script to work around TLS 1.2 requirement.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* See dotnet/coreclr#16868 and dotnet/versions#263

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
I'll make the update in versions repo shortly to use this new script.

File: UpdateDependencies.ps1 [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+$toolsLocalPath = Join-Path $PSScriptRoot "Tools"
+
+$initTools = Join-Path $PSScriptRoot "init-tools.cmd"
+& $initTools
+
+# Execute MSBuild using the dotnet.exe host
+$dotNetExe = Join-Path $toolsLocalPath "dotnetcli\dotnet.exe"
+$msbuildExe = Join-Path $toolsLocalPath "msbuild.exe"
+$testsBuildProj = Join-Path $PSScriptRoot "tests\build.proj"
```
Comment (dagood) [inline UpdateDependencies.ps1]: Ah, forgot about this--`tests\build.proj` is just `build.proj` here. I should have linked the CoreFX PR: https://github.com/dotnet/corefx/pull/27987. It replaces these lines with:

```
$updateProj = Join-Path $PSScriptRoot "build.proj"
& $dotNetExe $msbuildExe $updateProj /t:UpdateDependenciesAndSubmitPullRequest $args
```

**Comment by StephenBonikowsky** [comment]
Somehow have some other changes in here that shouldn't be here.
Need to clean it up.

**Comment by StephenBonikowsky** [comment]
My appologies, it was easier to just create a new PR.
#2676

**Comment by dagood** [comment]
Ah, it looked like an auto-update, so I assumed you run the script and included the changes it made in the commit. No problem.

**Pull request closed without merging**